app/src/main/java/com/onboarding/wheatherapp/mvp/presenter/WeatherAppPresenter.kt

```diff
@@ -1,14 +1,23 @@
 package com.onboarding.wheatherapp.mvp.presenter
 
 import com.onboarding.wheatherapp.mvp.WeatherAppContract
+import io.reactivex.rxjava3.android.schedulers.AndroidSchedulers
+import io.reactivex.rxjava3.schedulers.Schedulers
 
 class WeatherAppPresenter(
     private val model: WeatherAppContract.Model,
     private val view: WeatherAppContract.View
 ) : WeatherAppContract.Presenter {
 
+
     override fun getFiveDays() {
-        model.getData()
+        model.getData(CITY).subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
+            .subscribe { data ->
+                val aux = data
```

Comment:
i know you are not implementing the display, but the call to `view.showData(data)` should be here

Reply:
Solved!